easybuild/tools/run.py

```diff
@@ -227,7 +227,7 @@ def check_answers_list(answers):
     else:
         runLog = None
 
-    maxHitCount = 50
+    maxHitCount = maxhits 
```

Comment:
do you mind renaming this to `max_hit_count` while you're at it (we're trying to get rid of CamelCase throughout the whole codebase)?

or rather, just use `maxhits` wherever `maxHitCount` is used now, there's no need to define another variable for the same thing (since the value doesn't change afaik?)